insights/core/__init__.py

```diff
@@ -60,7 +60,7 @@ class Parser(object):
     """
 
     def __init__(self, context):
-        self.file_path = context.path
+        self.file_path = os.path.join("/", context.relative_path) if context.relative_path is not None else None
```

Comment:
Should we just add a `self.relative_path` to `Parser` and leave the `self.file_path` as it was?

Reply:
Originally, we think keep these two. But when I do the update. I think file_path and relative_path that will confuse others that consume the parser. In fact, the file_path in context is just exact temp dir, to evaluate one same archive two times, the value of file_path will be different also. So do not keep the value.  just keep relative_path as path here.

The parser

Reply:
Yep, currently, I also cannot imagine what the path that contains the temp dir can be used for yet. That makes sense to me.